### PR TITLE
Real time validation

### PR DESCRIPTION
**Pull request by delaneysmith22**
This PR contains changes that relate to the real-time validation of our A/B classifier using the WaveBase device. Specifically, changes have been made that enable us to: 
1. Build a frame dataset from the masked clips exported from the device and their corresponding Labelbox annotations, 
2. Make predictions on this dataset using the onnx version of the model,
3. Generate clip-level predictions from the frame-level prediction probability csv's exported from the device, and
4. Confirm preprocessing for inference on the WaveBase device (deploy.py)

### CODE REVIEW

File: src/predict.py [hunk 1]
```diff
@@ -282,6 +316,59 @@ def predict_with_contiguity_threshold(pred_probs, contiguity_threshold, classifi
     clip_pred = int(max_contiguous_b_line_preds(b_preds) >= contiguity_threshold)
     return np.array([1 - clip_pred, clip_pred])
 
+def predict_with_contiguity_threshold_wb(preds, target_class, contiguity_threshold, classification_threshold):
```
Comment (bvanberl) [inline src/predict.py:319]: Wondering if we should rename the function to emphasize that we're doing _clipwise_ predictions.

**Review by bvanberl — COMMENTED** [review]
Would you mind adding an entry for `deploy.py` in the [Project Structure](https://github.com/deepbreathe-ai/ab-line-classifier#project-structure) section of the readme?

**Review by bvanberl — APPROVED** [review]

File: src/deploy.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    resized_image = cv2.resize(image[0], INPUT_SIZE, interpolation=cv2.INTER_LINEAR)
+    resized_image = resized_image.reshape((1, INPUT_SIZE[0], INPUT_SIZE[1], N_CHANNELS))
+
+    # Apply scaling function
+    preprocessed_image = preprocessing_fn(resized_image)
+    return preprocessed_image
+
+
+def predict_wavebase_mp4(model_path, mp4_path, preds_path):
+
```
Comment (marwanrahman) [inline src/deploy.py:37]: Would be nice to add a docstring for this function

Comment (delaneysmith22) [reply src/deploy.py]: Done!

**Pull request merged**